ui/installer/VCSA/install.sh

```diff
@@ -131,30 +153,66 @@ upload_packages () {
         fi
     done
 
-    echo "-------------------------------------------------------------"
-    echo "Copying plugin contents to the vSphere Web Client..."
-    echo "Please enter the root password for your machine running VCSA"
-    echo "-------------------------------------------------------------"
-    scp -qr $PLUGIN_BUNDLES root@$VCENTER_IP:/tmp/
-    if [[ $? > 0 ]] ; then
-        echo "Error! Could not upload the VIC plugins to the target VCSA"
+    if [[ $BASH_ENABLED_ON_VCSA -eq 1 ]] ; then
+        echo "-------------------------------------------------------------"
+        echo "Copying plugin contents to the vSphere Web Client..."
+        echo "Please enter the root password"
+        echo "-------------------------------------------------------------"
+        scp -qr $PLUGIN_BUNDLES root@$VCENTER_IP:/tmp/
+        if [[ $? > 0 ]] ; then
+            echo "Error! Could not upload the VIC plugins to the target VCSA"
+            exit 1
+        fi
+    else
+        echo "-------------------------------------------------------------"
+        echo "WARNING!! Bash shell is required to complete the installation"
+        echo "automatically, however it is not enabled on this machine."
+        echo "Installation cannot be completed manually. Please follow the"
```

Comment:
Should this be `can be completed manually` or `cannot be completed automatically`?